src/components/nav/MusicBar2.vue

```diff
@@ -9,231 +9,253 @@
     :retain-focus="false"
     content-class="music-player-bar"
   >
-    <v-slider
-      class="music-progress"
-      hide-details
-      :value="progress"
-      height="3"
-      @change="progressChange"
-    />
-    <div
-      class="d-flex justify-space-between pa-2"
-      :class="{ 'flex-column': $vuetify.breakpoint.xs }"
-    >
-      <div class="player-controls d-flex align-center">
-        <div>
-          <v-btn
-            icon
-            class="mx-1"
-            color="secondary"
-            @click="prevButtonHandler"
-          >
-            <v-icon>{{ mdiSkipPrevious }}</v-icon>
-          </v-btn>
-          <v-btn
-            icon
-            fab
-            color="primary"
-            @click="playPause"
-          >
-            <v-icon x-large>
-              {{ playButtonIcon }}
-            </v-icon>
-          </v-btn>
-          <v-btn
-            icon
-            class="mx-1"
-            color="secondary"
-            @click="nextButtonHandler"
-          >
-            <v-progress-circular
-              color="warning"
-              :class="{ invisible: !showPatience }"
-              :value="patience"
-              size="40"
+    <div class="mouseEnterTrigger">
+      <v-slider
+        class="music-progress"
+        hide-details
+        :value="mathFloorTime()"
+        min="0"
+        :max="songlength"
+        :thumb-label="true"
+        height="3"
+        @change="progressChange"
+      >
+        <template #thumb-label="{ value }">
+          {{ timeFormat(value) }}
```

Comment:
this does not seem to be showing up anywhere...

Reply:
actually it shows up when i interact with the slider, but the value is not where the user is dragging it to and the UI element _FIGHTS_ the drag motion.